bot/exts/utils/snekbox.py

```diff
@@ -390,18 +473,26 @@ async def run_job(
         log.info(f"Received code from {ctx.author} for evaluation:\n{code}")
 
         while True:
-            self.jobs[ctx.author.id] = datetime.datetime.now()
+            self.jobs[ctx.author.id]["has_running_job"] = True
+
             try:
-                response = await self.send_job(ctx, code, args=args, job_name=job_name)
+                response = await self.send_job(ctx, python_version, code, args=args, job_name=job_name)
             finally:
-                del self.jobs[ctx.author.id]
-
-            code, args = await self.continue_job(ctx, response, ctx.command)
+                self.jobs[ctx.author.id]["has_running_job"] = False
+                # Store info about this job, to deal with the existence of the button that could
+                self.jobs[ctx.author.id][ctx.message.id] = {
+                    "code": code,
+                    "python_version": python_version,
+                    "start_date": datetime.datetime.now(),
```

Comment:
Where is this datetime ever used?

Reply:
It wasn't, it must have been a sentinel value previously and I thought it was used somewhere. removed in https://github.com/python-discord/bot/pull/2218/commits/0817398c8d05a16d9a0d1ab272e66ef9858f35c8